widgets-intro.md

```diff
@@ -469,17 +469,19 @@ class ShoppingListItem extends StatelessWidget {
   }
 
   TextStyle _getTextStyle(BuildContext context) {
-    if (inCart) {
-      return DefaultTextStyle.of(context).copyWith(
-          color: Colors.black54, decoration: TextDecoration.lineThrough);
-    }
-    return null;
+    if (!inCart)
+      return null;
+
+    return new TextStyle(
```

Comment:
Right. I was pleased to be able to remove the DefaultTextStyle copyWith() code. Not something you want to see in an intro :-).